.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+nmHoistingLimits: dependencies
```

Comment:
This is needed because serverless does not work well with the dependency structure `yarn workspaces` create